enterprise-ws-spec/src/main/asciidoc/Deployment.adoc

```diff
@@ -306,31 +302,23 @@ Support for type mappings is provider specific. There is no means for
 creating portable type mappings and therefore no means for declaring
 them or deploying them required by this specification.
 
-==== Mapping requirements
```

Comment:
would it make sense to keep this section and saying "No specific requirements." (or sth like that) instead?

Reply:
I will fix the broken links.  I don't know new Jakarta name for "Java 2 Security permissions", Is it "Jakarta Security permissions?  I think it's better to remove the "Mapping requirements" section as it is only specific to JAX-RPC and I have removed JAX-RPC only sections in other chapters.

Reply:
I agree with removing this section since it only applies to JAX-RPC.  "Java 2 Security permission" name has not changed - it is still part of Java SE.

Reply:
the [security guide](https://docs.oracle.com/javase/8/docs/technotes/guides/security/permissions.html) talks just about permissions. When I see 'Java 2' somewhere, my mind is getting back to J2SE 1.4. 'Java SE...', 'Java...', 'security permissions' or even just 'permissions' sounds much better...